### PR TITLE
Add PSA variables description, advanced, hidden information

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Review by ImenBizid — APPROVED** [review]

File: COCOAnnotator/resources/catalog/COCO_Annotator.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="COCO_Annotator" onTaskError="continueJobExecution" priority="normal" projectName="Service Automation - Deployment" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable model="" name="INSTANCE_NAME" value="coco-annotator"/>
-    <variable name="SERVICE_PORT" value="-1" model="PA:Integer"/>
-    <variable model="PA:Boolean" name="PROXYFIED" value="True"/>
-    <variable model="" name="ENDPOINT_ID" value="coco-annotator-gui"/>
-    <variable model="PA:Boolean" name="HTTP_ENABLED" value="True"/>
+    <variable model="" name="INSTANCE_NAME" value="coco-annotator-$PA_JOB_ID" description="The name of the service to be deployed." advanced="false" hidden="false"/>
```
Comment (fviale) [inline COCOAnnotator/resources/catalog/COCO_Annotator.xml:4]: can you please remove the following sentences in the workflow description as you add descriptions to the variables ?

`The service can be started using the following variables:
INSTANCE_NAME (Required): service instance name`

This should be done in all other workflows

File: Elasticsearch/resources/catalog/Elasticsearch.xml [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
   <variables>
-    <variable name="INSTANCE_NAME" value="elasticsearch-server" />
-    <variable name="PROXYFIED" value="True" model="PA:Boolean"/>
-    <variable name="ENDPOINT_ID" value="elasticsearch-endpoint-${PA_JOB_ID}" model=""/>
-    <variable name="HTTP_ENABLED" value="True" model="PA:Boolean"/>
+    <variable name="INSTANCE_NAME" value="elasticsearch-server-${PA_JOB_ID}" description="The name of the service to be deployed." advanced="false" hidden="false"/>
+    <variable name="PROXYFIED" value="True" model="PA:Boolean" description="True if a proxy is needed to protect the access to the service endpoint." advanced="true" hidden="false"/>
+    <variable name="ENDPOINT_ID" value="elasticsearch-endpoint-${PA_JOB_ID}" model="" description="The endpoint identifier that will be used." advanced="false" hidden="true"/>
+    <variable name="HTTP_ENABLED" value="True" model="PA:Boolean" description="True if an http endpoint need to be exposed to a user quick access to the service endpoint." advanced="false" hidden="true"/>
```
Comment (fviale) [inline Elasticsearch/resources/catalog/Elasticsearch.xml:9]: True if a http endpoint will be exposed as the service endpoint.

**Review by andrewssobral — APPROVED** [review]
LGTM!

**Pull request merged**